### PR TITLE
Publisher app global admin flag

### PR DESCRIPTION
**Pull request by awais786**
Create the is_global_admin flag in UserAttributes table. 
User having this value will be a super-user for publisher app.
Update the context processor for this new field.

https://openedx.atlassian.net/browse/ECOM-5950

### CODE REVIEW

**Comment by awais786** [comment]
@ahsan-ul-haq @tasawernawaz @waheedahmed Please review.

**Review by clintonb — CHANGES_REQUESTED** [review]
Why the user attribute instead of a Django group?

**Comment by awais786** [comment]
@edx/ecommerce Please review this PR.

**Comment by awais786** [comment]
@clintonb This user will be the super user for the publisher app only. That's why add the field in UserAttributes table. This table will specifically hold the value for publisher app users.

@bderusha FYI

**Comment by clintonb** [comment]
@awais786 I get that; but, on previous pull requests you're using Django Guardian and groups. Now you've added a custom attribute. Using a custom attribute will require you to make custom handlers to process that attribute. If you use a Django group, you can take advantage of all of the functionality that already exists in Django with minimal custom work.

**Comment by clintonb** [comment]
@awais786 I wrote up a quick example of using a group: https://github.com/edx/course-discovery/commit/6ef5118c5faba4e2b9dc3c7d693e8915c705bb71

**Comment by awais786** [comment]
@clintonb commits looks good. I will update my PR that way.

File: course_discovery/apps/publisher/migrations/0013_create_admin_group.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+ADMIN_GROUP_NAME = 'Publisher Admins'
```
Comment (bderusha) [inline course_discovery/apps/publisher/migrations/0013_create_admin_group.py:6]: Can we import this from `constants`?

File: course_discovery/apps/publisher/utils.py [hunk 2]
```diff
@@ -14,3 +15,15 @@ def is_email_notification_enabled(user):
         return user.attributes.enable_email_notification
 
     return True
+
+
+def is_global_admin(user):
```
Comment (bderusha) [inline course_discovery/apps/publisher/utils.py:20]: can we call this `is_publisher_admin`

Comment (bderusha) [reply course_discovery/apps/publisher/utils.py:20]: and rename the appropriate tests

**Pull request closed without merging**